src/codecs/pnm/decoder.rs

```diff
@@ -697,8 +755,8 @@ impl Sample for PbmBit {
         Ok((linelen * height) as usize)
     }
 
-    fn from_bytes(bytes: &[u8], row_size: u32, output_buf: &mut [u8]) -> ImageResult<()> {
-        let mut expanded = utils::expand_bits(1, row_size, bytes);
+    fn from_bytes(bytes: &[u8], row_size: usize, output_buf: &mut [u8]) -> ImageResult<()> {
+        let mut expanded = utils::expand_bits(1, row_size.try_into().unwrap(), bytes);
```

Comment:
This `unwrap` was added due to the type change. It should be explained, in a comment or by using `expect`, why this can not panic—otherwise this is seems to make stability worse than the panic-in-debug from multiplication. An alternative would be to return an error here, the result type is already available for this.